.github/workflows/ci_benchmark.yml

```diff
@@ -0,0 +1,150 @@
+
+      - name: "Prepare ccache"
+        id: prepare-ccache
+        shell: bash -l {0}
+        run: |
+          echo "key=benchmark-$RUNNER_OS" >> $GITHUB_OUTPUT
+          echo "timestamp=$(date +%Y%m%d-%H%M%S)" >> $GITHUB_OUTPUT
+          ccache -p
+          ccache -z
+
+      - name: "Restore ccache"
+        uses: actions/cache@v4
+        with:
+          path: .ccache
+          key: ccache-${{ secrets.CACHE_VERSION }}-${{ steps.prepare-ccache.outputs.key }}-${{ steps.prepare-ccache.outputs.timestamp }}
+          restore-keys: |
+            ccache-${{ secrets.CACHE_VERSION }}-${{ steps.prepare-ccache.outputs.key }}-
+
+      - name: Run benchmarks
+        shell: bash -l {0}
+        id: benchmark
+        env:
+          OPENBLAS_NUM_THREADS: 1
+          MKL_NUM_THREADS: 1
+          OMP_NUM_THREADS: 1
+          ASV_FACTOR: 1.3
+          ASV_SKIP_SLOW: 1
+        run: |
+          set -x
+          python -m pip install asv virtualenv packaging
+
+          git clone -b main https://github.com/astropy/astropy-benchmarks.git --single-branch
+
+          # ID this runner
+          python -m asv machine --yes --conf asv.ci.conf.json
+
+          echo "Baseline:  ${{ github.event.pull_request.base.sha }} (${{ github.event.pull_request.base.label }})"
+          echo "Contender: ${GITHUB_SHA} (${{ github.event.pull_request.head.label }})"
+
+          # Run benchmarks for current commit against base
+          ASV_OPTIONS="--split --show-stderr --factor $ASV_FACTOR --conf asv.ci.conf.json"
+          python -m asv continuous $ASV_OPTIONS ${{ github.event.pull_request.base.sha }} ${GITHUB_SHA} \
+              | sed "/Traceback \|failed$\|PERFORMANCE DECREASED/ s/^/::error::/" \
+              | tee benchmarks.log
+
+      - name: "Check ccache performance"
+        shell: bash -l {0}
+        run: ccache -s
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: asv-benchmark-results
+          path: benchmarks.log
+
+      - name: Download artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: asv-benchmark-results
+
+      - name: Read artifact
+        id: artifact
+        run: |
+          echo "{content}=$(cat benchmarks.log)" >> $GITHUB_OUTPUT
+
+      - name: Create comment
+        uses: actions/github-script@v7
+        if: github.event_name == 'pull_request_target'
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const issue_number = context.payload.pull_request.number;
+            const body = `
+            <details>
+              <summary>Benchmark results</summary>
+              \`\`\`
+              ${{ steps.artifact.outputs.content }}
+              \`\`\`
+            </details>
+            `;
+            github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issue_number,
+              body: body
+            });
```

Comment:
`pull_request_target` is not an event this workflow is subscribed to, so this will never run. Might as well delete it?

```suggestion
```

Reply:
Sure we can remove this for now.